riscv/insns/csrrc.h

```diff
@@ -1,8 +1,16 @@
 bool write = insn.rs1() != 0;
 int csr = validate_csr(insn.csr(), write);
-reg_t old = p->get_csr(csr);
-if (write) {
-  p->set_csr(csr, old & ~RS1);
+reg_t old;
+if (check_vcsr(csr)) {
```

Comment:
This file should not be modified, because demultiplexing CSRs shouldn't happen here.  Instead, the `get_csr` and `set_csr` routines should be modified.  This avoids repetition (as currently this logic is duplicated in 6 files).

Reply:
OK. We'll rollback.

Reply:
done